### PR TITLE
PyPI setup.

### PR DESCRIPTION
**Pull request by kattni**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        'Topic :: System :: Hardware',
+        'License :: OSI Approved :: MIT License',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.4',
+        'Programming Language :: Python :: 3.5',
+    ],
+
+    # What does your project relate to?
+    keywords='adafruit temperature barometric pressure temp'
+             'bmp3xx bmp388 hardware micropython circuitpython',
```
Comment (brennen) [inline setup.py:56]: Is this going to mash `temp` and `bmp3xx` together without an intervening space?

Comment (brennen) [reply setup.py:56]: Seems like:

```
>>> def foo(bar):
...   print(bar)
... 
>>> foo(bar='baz'
...     'bif')
bazbif
```

**Review by brennen — CHANGES_REQUESTED** [review]
Aside from that minor keyword string tweak that I've never thought about before, all looks good.

File: setup.py (hunk 1 above)
Comment (kattni) [reply setup.py:56]: I feel like I've done it this way before. I have no idea whether it's borked in those cases too. I'll add the space.

**Pull request merged**